.gitignore

```diff
@@ -1,23 +0,0 @@
-# Compiled class file
```

Comment:
You should have at least one `.gitignore` file in the root directory. Here is a template if you are working with IntelliJ: https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore

Reply:
I also like very much [this website to generate a gitignore file](https://www.toptal.com/developers/gitignore) for you. E.g. this one should work for you: https://www.toptal.com/developers/gitignore?templates=java,intellij,windows,linux,macos

Reply:
You can also watch those videos about how to use a `.gitignore` file
- https://youtu.be/4ukhZvOmAtk?t=590
- https://www.youtube.com/watch?v=6vvUevxlOp8

and also take a look at the preparation for the classes block in teaching block 1. There is a link to a page, which explains what the `.gitignore` file is -> https://rit-moodle.inso.tuwien.ac.at/mod/url/view.php?id=816

Reply:
> I also like very much [this website to generate a gitignore file](https://www.toptal.com/developers/gitignore) for you. E.g. this one should work for you: https://www.toptal.com/developers/gitignore?templates=java,intellij,windows,linux,macos

Thank you for sharing the link to generate a .gitignore file with just one click.

Reply:
> You can also watch those videos about how to use a `.gitignore` file
> 
> * https://youtu.be/4ukhZvOmAtk?t=590
> * https://www.youtube.com/watch?v=6vvUevxlOp8
> 
> and also take a look at the preparation for the classes block in teaching block 1. There is a link to a page, which explains what the `.gitignore` file is -> https://rit-moodle.inso.tuwien.ac.at/mod/url/view.php?id=816

Thank you for sharing the helpful resources on using a .gitignore file. Those were really helped.

Reply:
I understand it now clearly.  Thank you.